tests/frontend/systolic/array-1.expect

```diff
@@ -197,16 +269,37 @@ component main() -> () {
           }
           static par {
             incr_idx;
-            idx_between_8_9_group;
-            idx_between_0_3_group;
-            idx_between_1_4_group;
-            idx_between_5_8_group;
+            lt_iter_limit_group;
+            idx_between_5_depth_plus_5_group;
+            idx_between_0_depth_plus_0_group;
+            idx_between_1_depth_plus_1_group;
+            idx_between_1_min_depth_4_plus_1_group;
+            idx_between_depth_plus_5_depth_plus_6_group;
+            depth_plus_5_group;
+            depth_plus_0_group;
+            depth_plus_1_group;
+            min_depth_4_plus_1_group;
+            depth_plus_6_group;
           }
         }
       }
     }
   }
 }
+component main() -> () {
+  cells {
+    systolic_array = systolic_array_comp();
+    @external t0 = std_mem_d1(32, 3, 2);
+    @external l0 = std_mem_d1(32, 3, 2);
+    @external out_mem_0 = std_mem_d1(32, 1, 32);
+  }
+  wires {
+
+  }
+  control {
+    invoke systolic_array(depth=32'd3, t0_read_data=t0.read_data, l0_read_data=l0.read_data)(t0_addr0=t0.addr0, l0_addr0=l0.addr0, out_mem_0_addr0=out_mem_0.addr0, out_mem_0_write_data=out_mem_0.write_data, out_mem_0_write_en=out_mem_0.write_en);
```

Comment:
Can the various memories be passed by reference here?

Reply:
We probably could. But then I think the `systolic_array_comp` would have to have `ref cell std_mem_d2(//dimensions here);` declared as one of its cells, meaning that the component loses the flexibility of being able to multiply matrices with different contraction dimensions.

Reply:
Ah right, that's why we decided on this interface in the first place!